setup.py

```diff
@@ -19,5 +19,5 @@
     packages=[
         "niceplots",
     ],
-    install_requires=[],
+    install_requires=["numpy", "matplotlib"],
```

Comment:
This repo will probably work fine for other version of numpy/matplotlib but do we want to require a minimum version since we are testing for a minimum version? This could be,
```
install_requires=["numpy>=1.16", "matplotlib>=3.2"]
```
or similar.

Reply:
I think at the moment niceplots will work even with python 2 and mpl 2 etc so even though I think we shouldn't bother testing against those versions, I also don't think we should make the package incompatible with them for no reason

Reply:
Right, Eirikur is probably saying it would be good to figure out what the minimum requirements are instead of not putting in any version requirement, since presumably really old versions would not work.

Reply:
Sure, I think mpl 2.2.5 makes sense since niceplots uses the `constrained_layout` feature which was added in v2.2 and mpl also lists 2.2.* as a LTS version for py2.